sdl-core/src/main/scala/io/smartdatalake/workflow/action/ProxyAction.scala

```diff
@@ -40,7 +40,7 @@ import org.apache.spark.sql.types.StructType
  * If the execution of @wrappedAction is successful, the ProxyAction will return an empty SparkSubFeed by the correct schema.
  *
  */
-case class ProxyAction(wrappedAction: Action, override val id: SdlConfigObject.ActionId, agent: Agent) extends Action {
+case class ProxyAction(wrappedAction: Action, override val id: SdlConfigObject.ActionId, agent: Agent) extends Action with ExcludeFromSchemaExport {
```

Comment:
Very nice! We actually had a confused user in the SDL Course that found the ProxyAction in the Schema Viewer... Good that it gets hidden now

Reply:
it also caused problems in schema export because of the recursion it creates...